haystack/nodes/other/shaper.py

```diff
@@ -0,0 +1,287 @@
+import copy
+import inspect
+from collections import deque
+from typing import Optional, Union, List, Dict, Any, Tuple, Callable
+from types import MappingProxyType
+
+from haystack.nodes.base import BaseComponent
+from haystack.schema import Document, MultiLabel
+
+
+class Shaper(BaseComponent):
+    outgoing_edges = 1
+
+    def __init__(self, inputs: Dict[str, Any]):
+        super().__init__()
```

Comment:
I think it would be great if we could add a docstring explaining what the Shaper is for and then also a param docstring for `inputs`.